src/Humanizer/InflectorExtensions.cs

```diff
@@ -49,7 +60,7 @@ public static string Pluralize(this string word, bool inputIsKnownToBeSingular =
         /// <param name="inputIsKnownToBePlural">Normally you call Singularize on plural words; but if you're unsure call it with false</param>
         /// <param name="skipSimpleWords">Skip singularizing single words that have an 's' on the end</param>
         /// <returns></returns>
-        public static string Singularize(this string word, bool inputIsKnownToBePlural = true, bool skipSimpleWords = false)
+        public static string Singularize(this string word, bool inputIsKnownToBePlural = true, bool skipSimpleWords)
```

Comment:
The order here can't work, optional parameters have to be after non-optional ones -- this is causing the build to break.